daphne_worker/src/config.rs

```diff
@@ -538,16 +516,3 @@ impl AsRef<DapTaskConfig> for GuardedDapTaskConfig<'_> {
         self.value()
     }
 }
-
-/// Deployment types for Daphne-Worker. This defines overrides used to control inter-Aggregator
```

Comment:
Hmm, I'm definitely OK with removing `Dev` (and simplifying accordingly, as you have done), but removing this enum altogether is a regression, as I plan to have other deployment types, such as one that allows both Aggregators to run on Cloudflare infra.

For the sake of unblocking interop testing, let's leave as-is. I'll just leave a comment here to link to an issue.